bindings/python/iota_sdk/types/signature.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2023 IOTA Stiftung
+# SPDX-License-Identifier: Apache-2.0
+
+from iota_sdk.types.common import HexStr
+
+class Ed25519Signature():
+    def __init__(self, public_key: HexStr, signature: HexStr):
+        """Initialize an Ed25519Signature
+
+        Parameters
+        ----------
+        public_key : string
+            The Ed25519 public key.
+        signature : string
+            The Ed25519 signature.
```

Comment:
I think this is useless since we have the type hints and the naming should be enough
If you think is should stay, string should be HexStr instead